resources/views/inputs/includes/tile-selection-option.blade.php

```diff
@@ -21,14 +21,24 @@ class="hidden"
             wire:model="{{ $wireModel }}"
         />
     @else
-        <input
-            id="{{ $id.'-'.$option['id'] }}"
-            name="{{ $option['id'] }}"
-            type="checkbox"
-            class="form-checkbox tile-selection-checkbox"
-            x-model="options['{{ $option['id'] }}'].checked"
-            wire:model="{{ $wireModel }}"
-        />
+        {{--TODO: Present in the HTML but not displayed, maybe because of the disabled of the input ?--}}
+        <div
+            @if($isDisabled)
+                data-tippy-content="{{ $disabledCheckboxTooltip }}"
+            @endif
```

Comment:
`point-events-none` on the parent element means you will not trigger this

Reply:
Ohhh, good catch